modules/swagger-codegen/src/main/resources/JavaPlayFramework/newApiController.mustache

```diff
@@ -40,7 +44,7 @@ public class {{classname}}Controller extends Controller {
 {{#operation}}
 
     {{#wrapCalls}}@ApiAction{{/wrapCalls}}
-    public Result {{operationId}}({{#pathParams}}{{>pathParams}}{{#hasMore}},{{/hasMore}}{{/pathParams}}) {{^handleExceptions}}{{#bodyParams}}throws IOException{{/bodyParams}}{{/handleExceptions}}{{#handleExceptions}}throws Exception{{/handleExceptions}} {
+        public CompletationStage<Result> {{operationId}}({{#pathParams}}{{>pathParams}}{{#hasMore}},{{/hasMore}}{{/pathParams}}) {{^handleExceptions}}{{#bodyParams}}throws IOException{{/bodyParams}}{{/handleExceptions}}{{#handleExceptions}}throws Exception{{/handleExceptions}} {
```

Comment:
I will need to test this in my own project to be sure it has no side effect. I'll keep you updated.